### DISCUSSION TITLE
How to make component non-interactive when editing props

### DISCUSSION THREAD

**Discussion by vincenteof** [prevwong/craft.js#638] [Q&A]
For an example, I have a Button component to edit, and I can set some onClick handler to it. Is there any recommended way to make the handler not being triggered when editing props?

**Comment by alihammad99** [comment]
I don't know if this helps but I use it

```
const { id } = useNode();
  const { isActive } = useEditor((_, query) => ({
    isActive: query.getEvent("selected").contains(id),
  }));
  
  if (isActive) {
       // block other action ...
  }
```
